website/docs/how-to/how-to-troubleshoot-flag-not-returned.md

```diff
@@ -0,0 +1,14 @@
+---
+title: My feature flag is not returned in the Frontend API/Edge/Proxy
+---
+
+By default, these endpoints will not return features that are not enabled. This is mainly to save on bandwidth but it can be confusing when you expect your customers to be exposed to a feature but they’re not. What features are accessible to you is mainly determined by the token you’re using and the type of access it has.
```

Comment:
I know we're inconsistent about this other places in the docs, but are they features or feature flags? Might not be worth changing other places, but it may provide additional clarity here? 🤔 

```suggestion
By default, these endpoints will not return feature flags that are not enabled. This is mainly to save on bandwidth but it can be confusing when you expect your customers to be exposed to a feature but they’re not. What features are accessible to you is mainly determined by the token you’re using and the type of access it has.
```